docs/advanced-features/event-driven-packaging.md

```diff
@@ -0,0 +1,181 @@
+<!-- markdownlint-disable -->
+# Event-Driven Packaging (EDP)
+
+> EDP is currently in private preview. Please contact Quilt
+directly to discuss pricing.
+
+## Overview
+
+Data can be added to your Amazon S3 buckets manually by internal
+team members and/or external collaborators, or automatically via
+instrumentation. No matter how it arrives into your buckets, it 
+is helpful, and sometimes critical, to be notified of this data 
+migration.
+
+Quilt's *Event-Driven Packaging* service (EDP) monitors S3 object events
+for a given Amazon S3 bucket prefix and groups those S3 object
+events into logical batches via custom-defined time, space and
+prefix heuristics. A `files_ready` event is generated on the receiving
+[EventBridge](https://aws.amazon.com/eventbridge/) in order to
+notify users and/or other AWS services that this logical batch of
+S3 objects are ready to be turned into Quilt packages. *This makes
+it easy to ingest data into Quilt data packages from AWS data
+migration services* such as [Storage
+Gateway](https://aws.amazon.com/storagegateway/) and [Data
+Sync](https://aws.amazon.com/datasync/).
+
+> Any AWS service or action that generates S3 object events may trigger the EDP service.
+
+## Requirements
+
+1. A pre-existing VPC that includes a [NAT
+Gateway](https://docs.aws.amazon.com/vpc/latest/userguide/vpc-nat-gateway.html)
+or the following [VPC
+endpoints](https://docs.aws.amazon.com/vpc/latest/privatelink/concepts.html#concepts-vpc-endpoints):
+  - Amazon S3
+  ([gateway](https://docs.aws.amazon.com/vpc/latest/privatelink/gateway-endpoints.html)
+  or
+  [interface](https://docs.aws.amazon.com/vpc/latest/privatelink/create-interface-endpoint.html))
+  - EventBridge (interface)
+2. [Enable EventBridge S3
+Events](https://docs.aws.amazon.com/AmazonS3/latest/userguide/enable-event-notifications-eventbridge.html)
+for all S3 buckets to be monitored by EDP
+
+![](../imgs/edp-event-bridge-s3-properties.png)
+
+## Deployment
+
+To use EDP in your Quilt instance, you will need an additional
+CloudFormation template to configure when, and how, events get
+generated.
+
+### Parameters
+
+The following parameters need to be defined in the CloudFormation
+template during deployment:
+
+- `VPC`: Same as the existing Quilt instance.
+- `SecurityGroup`: Same as the existing Quilt instance.
+- `Subnets`: Same two subnets as the existing Quilt instance.
+- `BucketName`: Name of the Amazon S3 bucket to monitor. Must be
+read accessible to EDP.
```

Comment:
Does this mean anything? EDP is not a principal. Need to be more specific.

Reply:
- For a customer AWS admin they need to know that the `SecurityGroup` and `Subnets` are the same as the existing Quilt instance.
- I'll remove the `Must be read accessible....` sentence. Agree that doesn't make sense and makes it sound like EDP has a role or is a principal